tests/test_api/test_auth.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+
+from hyp3_api import auth
+
+
+def test_decode_asf_cookie(monkeypatch, jwks_client):
+    def mock_decode(*args, **kwargs):
+        return {'urs-user-id': 'test-user', 'urs-access-token': 'test-token'}
+
+    monkeypatch.setattr(auth.jwt, 'decode', mock_decode)
+
+    user, token = auth.decode_asf_cookie('test-token')
+    assert user == 'test-user'
+    assert token == 'test-token'
+
+
+@pytest.mark.network
+def test_decode_edl_bearer_token(monkeypatch, jwks_client):
+    def mock_decode(*args, **kwargs):
+        return {'uid': 'test-user'}
+
+    monkeypatch.setattr(auth.jwt, 'decode', mock_decode)
```

Comment:
It feels like we're mocking out a big chunk of the functionality by mocking `jwt.decode` in these two tests. I wonder how difficult it would be to come up with an actual token and cookie that could be decoded for these tests. Or would we run into an expiration date at some point?